src/Pickles/Pickles/Runner.cs

```diff
@@ -97,23 +97,38 @@ private static void SetResultsForIndividualScenariosUnderFeature(FeatureNode fea
 
                 if (scenario != null)
                 {
-                    featureElement.Result = testResults.GetScenarioResult(scenario);
+                    try { 
+                        featureElement.Result = testResults.GetScenarioResult(scenario);
+                    }
+                    catch (Exception e)
+                    {
+                        Log.Error("An exception occured in aligning results with the feature for : " +
+                            featureTreeNode.Feature.Name +
+                            " Message: " + e.Message);
+                    }
                     continue;
                 }
 
                 var scenarioOutline = featureElement as ScenarioOutline;
 
                 if (scenarioOutline != null)
                 {
-                    foreach (var example in scenarioOutline.Examples.SelectMany(e => e.TableArgument.DataRows))
+                    try
                     {
-                        example.Result = testResults.GetExampleResult(scenarioOutline, example.Cells.ToArray());
+                        foreach (var example in scenarioOutline.Examples.SelectMany(e => e.TableArgument.DataRows))
+                        {
+                            example.Result = testResults.GetExampleResult(scenarioOutline, example.Cells.ToArray());
+                        }
+
+                        scenarioOutline.Result =
+                            scenarioOutline.Examples.SelectMany(e => e.TableArgument.DataRows)
+                                .Select(row => row.Result)
+                                .Merge();
+                    } catch (Exception e) {
+                        Log.Error("An exception occured in aligning results with the feature for : " +
```

Comment:
Again: you're blindly logging an exception but taking no appropriate action. You should at the very least rethrow the exception. You should also be more selective in which exceptions you catch.